src/frontend/src/helpers/Compiler/OnlineCompiler.ts

```diff
@@ -100,4 +103,12 @@ class OnlineCompiler extends AbstractCompiler {
       pid: this.activePIDs[0]
     });
   }
+
+  protected programDone(pid: number) {
```

Comment:
The commit I just pushed should address this - forgot to make the change earlier